src/view/moveable/moveable.jsx

```diff
@@ -16,7 +16,7 @@ const origin: Position = {
 };
 
 const noMovement: Style = {
-  transform: null,
+  transform: undefined
```

Comment:
is it better just to remove the property?

Reply:
I had that thought too, but flow complained about `{}` not matching the shape, so I switched to this approach of just using `undefined`.